src/utilities/example-group/example-group.factory.ts

```diff
@@ -61,14 +49,82 @@ class ExampleGroupFactory {
       const defaultGroup = new ExampleGroup(
         operation,
         DEFAULT_PARAMETER_GROUP,
-        { ...requiredExamples, ...nonRequiredExamplesWithExampleField },
+        { ...requiredExamples, ...nonRequiredExamples },
       );
       exampleGroups.push(defaultGroup);
     }
 
     return exampleGroups;
   }
 
+  private static getNonRequiredExamples(parameters): { [name: string]: Json } {
+    return parameters
+      .filter((parameter): boolean => {
+        if (parameter.required) return false;
+
+        // Examples on parameters should always override examples found elsewhere
+        if (parameter.example) {
+          return Boolean(parameter.example);
+        }
+
+        if (parameterHasSchema(parameter)) {
+          return Boolean(parameter.schema.example);
+        }
+
+        if (parameterHasContent(parameter)) {
+          const [key] = Object.keys(parameter.content);
+
+          return Boolean(parameter.content[key].example);
+        }
+
+        return false;
+      })
+      .reduce((examples, parameter) => {
+        return {
+          ...examples,
+          [parameter.name]: parameter.example,
```

Comment:
what about when the example comes from the schema or content? The filter will return true, but this won't pull out the actual value